test/blip/basics/DailyDose.test.js

```diff
@@ -0,0 +1,234 @@
+/* global chai */
+/* global sinon */
+
+var React = require('react');
+var TestUtils = require('react-addons-test-utils');
+var expect = chai.expect;
+
+var DailyDose = require('../../../plugins/blip/basics/components/chart/DailyDose');
+
+describe('DailyDose', function () {
+
+  it('should be a function', function() {
+    expect(DailyDose).to.be.a('function');
+  });
+
+  describe('render', function() {
+    it('should render and show 2 warning messages for missing props', function () {
+      console.error = sinon.stub();
+      var elem = TestUtils.renderIntoDocument(<DailyDose/>);
+      expect(console.error.callCount).to.equal(2);
+    });
+
+    it('should render without problem when props provided', function () {
+      console.error = sinon.stub();
+
+      var props = {
+        data: {
+          weight: 100,
+          totalDailyDose: 11
+        },
+        addToBasicsData: sinon.stub()
+      };
+
+      var elem = TestUtils.renderIntoDocument(
+        <DailyDose 
+          data={props.data}
+          addToBasicsData={props.addToBasicsData}/>
+      );
+      
+      expect(console.error.callCount).to.equal(0);
+
+      // actual rendered text is modified version of input 'note'
```

Comment:
I'm super confused re: what you mean with this comment, which is in many places... Clarify?